src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -184,6 +185,14 @@ public void testLocalNodeRequest() {
         verify(transportService, times(1)).sendRequest(any(), anyString(), any(), any(LocalNodeResponseHandler.class));
     }
 
+    @Test
+    public void testActionListenerOnFailureRequest() {
+
+        extensionsRunner.sendActionListenerOnFailureRequest(transportService, new Exception("Test failure"));
+
+        verify(transportService, times(1)).sendRequest(any(), anyString(), any(), any(ActionListenerOnFailureResponseHandler.class));
```

Comment:
shouldn't we be able to test the value of the response?

Reply:
Resolved offline